core/conn/trafci/src/org/trafodion/ci/Properties/trafciDefaultLookAndFeel.properties

```diff
@@ -1,3 +1,26 @@
+#
```

Comment:
I thought we had put this file in the exception list ? Code won't be able to handle the copyright comments